daml-assistant/src/DA/Daml/Assistant/Cache.hs

```diff
@@ -53,15 +54,15 @@ cacheAvailableSdkVersions
     :: DamlPath
     -> CachePath
     -> IO [SdkVersion]
-    -> IO [SdkVersion]
+    -> IO ([SdkVersion], CacheAge)
 cacheAvailableSdkVersions damlPath cachePath getVersions = do
     damlConfigE <- tryConfig $ readDamlConfig damlPath
     let updateCheckM = join $ eitherToMaybe (queryDamlConfig ["update-check"] =<< damlConfigE)
         defaultUpdateCheck = UpdateCheckEvery (CacheTimeout 86400)
     case fromMaybe defaultUpdateCheck updateCheckM of
         UpdateCheckNever -> do
-            valueAgeM <- loadFromCacheWith cachePath versionsKey (CacheTimeout 0) deserializeVersions
-            pure (maybe [] fst valueAgeM)
+            valueAgeM <- loadFromCacheWith cachePath versionsKey (CacheTimeout 86400) deserializeVersions
```

Comment:
I’m confused by this. If we have `UpdateCheckNever` why not treat all values as stale like we did before?

Reply:
ah, no reason, this was a brain fart, reverted